### PR TITLE
CAN-1042 update the mongo tree for fixing the score update on support

### PR DESCRIPTION
**Pull request by sajid315**

### CODE REVIEW

File: app/Helpers/TopicSupport.php [hunk 1]
```diff
@@ -598,6 +642,12 @@ public static function reorderSupport($orders, $topicNum, $allNickNames)
                     ->whereIn('nick_name_id', $allNickNames)
                     ->update(['support_order' => $order['order']  // update your field(s) here
                         ]);
+                    
+                    /* To update the Mongo Tree while adding support - CAN-1042 */
+                    /* We need to update the execute the job for reordering as well, only when topicnumber == 81 */
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: I didn't understand this comment? What does it mean?

File: app/Helpers/TopicSupport.php [hunk 2]
```diff
@@ -587,6 +628,9 @@ public static function reorderSupport($orders, $topicNum, $allNickNames)
     {
 
         try{
+            /* To update the Mongo Tree while adding support - CAN-1042 */
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Remove the ticket number

File: app/Helpers/TopicSupport.php [hunk 3]
```diff
@@ -598,6 +642,12 @@ public static function reorderSupport($orders, $topicNum, $allNickNames)
                     ->whereIn('nick_name_id', $allNickNames)
                     ->update(['support_order' => $order['order']  // update your field(s) here
                         ]);
+                    
+                    /* To update the Mongo Tree while adding support - CAN-1042 */
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Remove the ticket number pls

**Review by akuks — CHANGES_REQUESTED** [review]

File: app/Helpers/TopicSupport.php (hunk 1 above)
Comment (sajid315) [reply app/Helpers/TopicSupport.php]: This comment means that we are updating mongo tree when user will add new support. 
And the second line means that there are multiple camp numbers here, so we are using the dynamic camp number for topic 81 only at canonizer service. thats why i mentioned this in comment.

File: app/Helpers/TopicSupport.php (hunk 2 above)
Comment (sajid315) [reply app/Helpers/TopicSupport.php]: Ok.

File: app/Helpers/TopicSupport.php (hunk 3 above)
Comment (sajid315) [reply app/Helpers/TopicSupport.php]: Ok

**Review by akuks — APPROVED** [review]

**Pull request merged**